### PR TITLE
184855662 - remove inactive filters

### PR DESCRIPTION
**Pull request by kamtarnowski**
#### What's this PR do?
Remove inactive filters and TOTAL stake reference over DC pages.

#### How should this be manually tested?
- verify filters in DC pages( index, show and ASN related versions)
- Total Stake was renamed into Stake. Please verify that.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184855662)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/views/asns/show.html.erb [hunk 1]
```diff
@@ -41,13 +41,6 @@
             <td><%= @population %></td>
           </tr>
           <tr>
-            <td><strong>Total Stake</strong></td>
```
Comment (kbiala) [inline app/views/asns/show.html.erb:44]: We do not want to remove this field, just rename it to "Stake" (see the screenshots in PT)

File: test/models/validator_test.rb [hunk 2]
```diff
@@ -183,11 +181,11 @@ class ValidatorTest < ActiveSupport::TestCase
     assert result.last.delinquent?
   end
 
-  test "#filtered_by inactive excludes correct validators from collection" do
-    result = Validator.filtered_by(["inactive"])
+  test "#filtered_by active excludes correct validators from collection" do
```
Comment (kbiala) [inline test/models/validator_test.rb:184]: Please fix this test name. Filtering by active (or anything else), doesn't exclude any validators. On the contrary, it includes all validators that meet condition (in this case - validators that are active), but they also may or may not meet other conditions (eg. might be also active and private) - it doesn't matter, as long as they meet selected filter. While I understand what test author wanted to say, this test description is confusing and incorrect.
It should be something like `#filtered_by active includes all active validators in collection`

File: test/models/validator_test.rb [hunk 3]
```diff
@@ -198,10 +196,10 @@ class ValidatorTest < ActiveSupport::TestCase
   end
 
   test "#filtered_by multiple parameters excludes correct validators from collection" do
```
Comment (kbiala) [inline test/models/validator_test.rb:198]: Same here. Should be something like `#filtered_by multiple parameters includes all validators that meet any of selected criteria"`

**Review by kbiala — CHANGES_REQUESTED** [review]

File: app/views/asns/show.html.erb (hunk 1 above)
Comment (kamtarnowski) [reply app/views/asns/show.html.erb:44]: Fixed.

**Review by kbiala — APPROVED** [review]

**Pull request merged**